numpy/lib/scimath.py

```diff
@@ -472,11 +472,18 @@ def power(x, p):
 
     >>> np.emath.power([2, 4], 2)
```

Comment:
Let's start with the simplest.
```suggestion
    >>> np.emath.power(2, 2)
    4
    
    >>> np.emath.power([2, 4], 2)
```